common/src/main/java/com/microsoft/identity/common/internal/request/MsalBrokerRequestAdapter.java

```diff
@@ -475,29 +499,44 @@ private Bundle getRequestBundleFromBrokerRequest(@NonNull BrokerRequest brokerRe
                     sRequestAdapterGsonInstance.toJson(brokerRequest, BrokerRequest.class)
             );
         }
+        requestBundle.putString(
+                AuthenticationConstants.Broker.NEGOTIATED_BP_VERSION_KEY,
+                negotiatedBrokerProtocolVersion
+        );
         return requestBundle;
     }
 
-    public Bundle getRequestBundleForGetAccounts(@NonNull final CommandParameters parameters) {
+    public Bundle getRequestBundleForGetAccounts(@NonNull final CommandParameters parameters,
```

Comment:
Just calling out that this change alters binary compatibility of `common`....

Reply:
Ack. these are only used by MSAL for these the request , so we should be good.